lib/value/src/value/serde.rs

```diff
@@ -29,19 +28,22 @@ impl Value {
         }
     }
 
-    // TODO: return Cow 🐄
     /// Converts self into a `String` representation, using JSON for `Map`/`Array`.
```

Comment:
Should the doc comment now refer to `Cow` ?

Reply:
I don't think it's a big deal. The Rust reference docs for `String::from_utf8_lossy` says "Converts a slice of bytes to a string" despite returning a `Cow`.